client/js/screens/Home/HomeContainer.js

```diff
@@ -1,15 +1,68 @@
 import React, {Component} from 'react';
+import {compose} from 'recompose';
+import {graphql, Query} from 'react-apollo';
+import {QUERY_USER, QUERY_EVENT_THIS_WEEK} from '../../apollo/queries';
+import {StatusBar, Text, View, LayoutAnimation} from 'react-native';
+import {VibrancyView} from '@react-native-community/blur';
 import Home from './Home';
+import {getParamFromParent} from '../../lib/paramFromParent';
 import PropTypes from 'prop-types';
 
 class HomeContainer extends Component {
+  static navigationOptions = props => {
+    return {
+      headerTransparent: true,
+    };
+  };
+
   render() {
-    const {navigation} = this.props;
-    return <Home navigation={navigation} />;
+    const {navigation, userInfo, eventThisWeek} = this.props;
+
+    return (
+      <Home
+        navigation={navigation}
+        userInfo={userInfo.user}
+        eventInfo={{
+          thisWeek: eventThisWeek.events,
+        }}
+      />
+    );
   }
 }
 
-export default HomeContainer;
+export default compose(
+  graphql(QUERY_USER, {
+    name: 'userInfo',
+    options: ({navigation}) => {
+      const userToken = getParamFromParent(navigation, 'userToken');
```

Comment:
You should just call the AsyncStorage to get the userToken, not the params.

Reply:
I can't do that in `compose()` function